service/linkAccount.js

```diff
@@ -0,0 +1,51 @@
+const promise = require('bluebird');
+const dbHelper = require('../helper/database');
+const config = require('../config');
+const message = config.message;
+const tokenCollection = 'token';
+
+let isTokenAlreadyUsed = (token) => {
+    return new promise(function (resolve, reject) {
```

Comment:
promise needs to be uppercase: new Promise(..//). Please change all over the place :)